openmdao/core/group.py

```diff
@@ -3196,8 +3196,14 @@ def _resolve_ambiguous_input_meta(self):
                     tmeta = abs2meta[tgt] if tgt in abs2meta else all_abs2meta[tgt]
                     tunits = tmeta['units'] if 'units' in tmeta else None
                     if 'units' not in gmeta and sunits != tunits:
-                        errs.add('units')
-                        metadata.add('units')
+                        try:
+                            test_conv = unit_conversion(sunits, tunits)
```

Comment:
Since PhysicalUnit objects understand if they're equivalent or not, we could do this a little more cleanly:

```
if _find_unit(sunits) != _find_unit(tunits):
   errs.add('units')
   metadata.add('units')
```

_find_unit is found in openmdao.utils.units